decidim-core/spec/lib/events/simple_event_spec.rb

```diff
@@ -20,7 +20,7 @@ module Decidim
 
       it "escapes the HTML tags from the i18n options" do
```

Comment:
We're changing this behaviour, right? 
```suggestion
      it "sanitizes the HTML tags from the i18n options" do
```